content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,122 @@
+---
+title: InfluxDB Cloud limits
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account limits per organization and hard global system limits. Review the following InfluxDB Cloud limits and errors to plan for your bandwidth needs:
+
+- [Account limits](#account-limits)
+- [Global limits](#global-limits)
+- [Limit errors](#limit-errors)
+
+<!--To estimate your projected usage costs, use the [InfluxDB Cloud pricing calculator](/influxdb/cloud/account-management/pricing-calculator/). -->
+
+## Account limits
+
+InfluxDB Cloud has adjustable limits applied per-account to reduce the chance of unexpected charges and protect the service for all users.
+
+_To request higher limits, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Rate limits
+
+{{% warn %}}
+Rate limits are accrued against a five-minute window. Rate limits for writes are applied to uncompressed payloads.
+{{% /warn %}}
+<!-- Include something about how the rate limit is calculated. -->
+
+#### Free Plan rate limits
+
+- Read: 300 MB data every 5 minutes
+- Write: 5.1 MB data every 5 minutes
+
+#### Usage-Based Plan rate limits
+
+- Read: 3 GB data every 5 minutes
+- Write: 3 GB data every 5 minutes
+
+### Other limits
+
+Other limits include all adjustable limits other than rate limits.
+
+#### Free Plan limits
+
+- Maximum of 10,000 series (see [cardinality](/influxdb/cloud/reference/glossary/#series-cardinality))
+- Available resources:
+  - 2 buckets
+  - 2 notification rules
+  - 5 dashboards
+  - 5 tasks
+  - 2 checks
+  - `http` and `pagerduty` notification endpoints
```

Comment:
Is Pagerduty now part of the free tier? This used to say "unlimited slack endpoints"?

Reply:
See also this conflicting information: https://docs.influxdata.com/influxdb/cloud/monitor-alert/notification-endpoints/create/